frontend/src/lib/components/PersonPropertySelect/PersonPropertySelect.tsx

```diff
@@ -23,53 +23,53 @@ export const PersonPropertySelect = ({ onChange, selectedProperties }: PersonPro
     }
 
     return (
-        <div>
-            <div>
-                {selectedProperties.length > 0 &&
-                    selectedProperties.map((name) => {
-                        return (
-                            <Tag
-                                key={name}
-                                closable
-                                onClose={(): void => handleRemove(name)}
-                                style={{
-                                    margin: '0.25rem',
-                                    padding: '0.25rem 0.5em',
-                                    background: '#D9D9D9',
-                                    border: '1px solid #D9D9D9',
-                                    borderRadius: '40px',
-                                }}
-                            >
-                                {name}
-                            </Tag>
-                        )
-                    })}
-
-                <Popup
-                    visible={open}
-                    overlay={
-                        <TaxonomicFilter
-                            onChange={(_, value) => {
-                                handleChange(value as string)
-                                hide()
+        <div style={{ marginBottom: 16 }}>
+            {selectedProperties.length > 0 &&
+                selectedProperties.map((name) => {
+                    return (
+                        <Tag
+                            key={name}
+                            closable
+                            onClose={(): void => handleRemove(name)}
+                            style={{
```

Comment:
non-blocking

I was thinking either creating our own Tag component which we can use everywhere, or pull this css into a css file, or both I guess.

It would be nice if we could somehow get rid of all the CSS noise.

Reply:
We can probably simply marry up and create components for each of @clarkus figma components

Reply:
Good idea

Reply:
Hmm, I'd do this in a follow up if that's okay - not quite sure where I should be defining this style, since components + custom tags (my select box) all will inherit from this O.o - maybe pass as params 🤷